pkg/strategy/grid2/strategy.go

```diff
@@ -405,6 +427,11 @@ func (s *Strategy) processFilledOrder(o types.Order) {
 
 		// use the profit to buy more inventory in the grid
 		if s.Compound || s.EarnBase {
+			// if it's not using the platform fee currency, reduce the quote quantity for the buy order
+			if feeCurrency == s.Market.QuoteCurrency {
+				orderExecutedQuoteAmount = orderExecutedQuoteAmount.Sub(feeQuantityReduction)
```

Comment:
What if the fee currency is base currency?

Reply:
it's sell triggers buy order, so we only care about the quote currency here?